frontend/helpers/product-list-helpers.ts

```diff
@@ -102,21 +102,14 @@ export function getProductListTitle(
    return productList.title;
 }
 
-export function getRefinementDisplayType(
-   attribute: string,
-   productListType: ProductListType
-) {
-   switch (attribute) {
-      case 'facet_tags.Item Type': {
-         switch (productListType) {
-            case ProductListType.DeviceParts:
-               return RefinementDisplayType.SingleSelect;
-            default:
-               return RefinementDisplayType.MultiSelect;
-         }
-      }
-      default: {
-         return RefinementDisplayType.MultiSelect;
-      }
-   }
+const refinementDisplayTypeMap: Partial<Record<string, RefinementDisplayType>> =
```

Comment:
Why is this partial?

Reply:
No good reason. I think it was an artifact of me playing around with the shape of this data structure.

Reply:
https://github.com/iFixit/react-commerce/pull/476/commits/7bf04ec8a13081c784bd5078ab709fff6b4b39c8